app/src/renderer/components/common/TmSessionImport.vue

```diff
@@ -2,15 +2,15 @@
   <div class="tm-session">
     <tm-form-struct :submit="onSubmit.bind(this)" class="tm-session-container">
       <div class="tm-session-header">
-        <a
-          @click="setState('welcome')"
-        >
+        <a @click="setState('welcome')">
           <i class="material-icons">arrow_back</i>
         </a>
         <div class="tm-session-title">
           Import with Seed
         </div>
-        <a @click="help"><i class="material-icons">help_outline</i></a>
+        <a @click="$store.commit(`toggleSessionModal`, false)">
```

Comment:
repetitive code

Reply:
yes. i couldn't come up with a simple way of addressing this without re-architecting the session modals / slots.

Reply:
Follow up https://github.com/cosmos/voyager/issues/2155